src/git.sh

```
@@ -1,42 +1,329 @@
+  local version=$(echo "$args_json" | jq -r '.version // ""')
+  local pr_title=$(echo "$args_json" | jq -r '.pr_title // ""')
+  local pr_message=$(echo "$args_json" | jq -r '.pr_message // ""')
+
+  git add .
+  git commit -am "chore: ${pr_title} ${pr_message}"
+  git push origin main
+
+  git_create_tag --version "$version" --tag_message "$pr_title"
+}
+
+git_create_tag() {
```

Shellcheck (0.10.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)